ruff.toml

```diff
@@ -47,6 +47,10 @@ ignore = [
 	"TRY400",
 ]
 
+[lint.isort]
+combine-as-imports = true
+split-on-trailing-comma = false
+
```

Comment:
Should these changes go into the skeleton? Why are they unique to this project?

Reply:
In fc15d4575, I've reverted this change and applied the standard formatting. I'd rather not tussle with ruff's preferences unless there's a strong reason to do so.